### PR TITLE
Add demo 2 and fix bug

### PR DESCRIPTION
**Pull request by whzzt**

### CODE REVIEW

File: demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,218 @@
+{
```
Comment (borontion) [inline demo.ipynb:1]: Could you move this to examples

File: docs/v0.1.0/frontend.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
+# v0.1.0 Frontend Document
```
Comment (borontion) [inline docs/v0.1.0/frontend.md:1]: I don't think we need versioning at this moment.  So just put everything under `docs/` should be fine.

File: examples/list_red_moving_vehicle/getcolor.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+import numpy as np
+import cv2
```
Comment (borontion) [inline examples/list_red_moving_vehicle/getcolor.py:3]: Could you dump your requirements file for each example?

File: examples/count_person/main.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
+"""This is a demo VQPy implementation listing and storing all red moving
```
Comment (borontion) [inline examples/count_person/main.py:1]: Should this comment be updated?

File: examples/list_red_moving_vehicle/getcolor.py (hunk 3 above)
Comment (borontion) [reply examples/list_red_moving_vehicle/getcolor.py:3]: And if VQPy has dependencies, you can use `install_requires` argument of the `setuptools.setup` function in `setup.py` file.

File: demo.ipynb (hunk 1 above)
Comment (yushan111) [reply demo.ipynb]: Done

File: docs/v0.1.0/frontend.md (hunk 2 above)
Comment (yushan111) [reply docs/v0.1.0/frontend.md]: Done

File: examples/list_red_moving_vehicle/getcolor.py (hunk 3 above)
Comment (yushan111) [reply examples/list_red_moving_vehicle/getcolor.py:3]: Not resolved yet. I have create an issue #7 for easy tracking

File: examples/count_person/main.py (hunk 4 above)
Comment (yushan111) [reply examples/count_person/main.py]: This comment has been removed in this PR. And an introduction will be included in example Readme instead.

**Comment by yushan111** [comment]
This PR will be merged first. We can create new PRs for further reviews.

**Review by yushan111 — APPROVED** [review]
Approve this PR for quick merge

**Pull request merged**